### PR TITLE
NR-31194 NumPy integration

### PR DESCRIPTION
**Pull request by pkosanam**
# Summary

Ticket URL: https://issues.newrelic.com/browse/NR-31194

Description:

This PR has the new quickstart integration named "NumPy"

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

**Comment by pkosanam** [comment]
Here is the IO page preview:
screencapture-newrelic-instant-observability-preview-2022-07-29-11_34_12

**Comment by aswanson-nr** [comment]
I'm assuming this one  is not stale!

File: quickstarts/python/numpy/config.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+slug: numpy
+title: NumPy
+
+description: |
+  ## Comprehensive monitoring quickstart for NumPy
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:6]:
```suggestion
  ## Comprehensive quickstart for NumPy
```

File: quickstarts/python/numpy/config.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  Monitor your NumPy applications, and analyze performance bottlenecks while operating in a live development environment.Full support for popular frameworks including Flask Visually monitor your web user experience across the world with your application rendition via dashboard.
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:8]:
```suggestion
With our NumPy integration, you can find and address performance bottlenecks while operating in a live development environment. Monitor your app via our dashboard quickstarts and ensure you're delivering the best user experience.
```

File: quickstarts/python/numpy/config.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  ## Why monitor NumPy?
+
+  An ideal NumPy monitoring tool can proactively capture, analyze, trace, and display information related to Python-developed applications.
+
+  An ideal NumPy Performance Monitor must offer comprehensive and actionable information that software developers need to troubleshoot python applications successfully. Some key components are:
+
+  - Get end-to-end visibility into your NumPy application environment and all the dependencies
+  - Reduce NumPy application downtime and detect the errors along with exceptions
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:17]:
```suggestion
  - Reduce NumPy application downtime and detect the errors and exceptions
```

File: quickstarts/python/numpy/config.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - Manage performance of homogeneous datasets and heterogeneous databases in context with your Numpy monitoring
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:18]:
```suggestion
  - Manage homogeneous and heterogeneous databases in context with your NumPy performance monitoring
```

File: quickstarts/python/numpy/config.yml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - Visually understand how a database is impacting your application
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:19]:
```suggestion
  - Get a visual overview of how a database affects your application
```

File: quickstarts/python/numpy/config.yml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  - Assess the visibility of the underlying server infrastructure in relation to your Numpy monitoring
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:20]:
```suggestion
  - Assess the visibility of the underlying server infrastructure in relation to your NumPy monitoring
```

File: quickstarts/python/numpy/config.yml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  - Compare the performance of your Numpy transactions
+  - Visually analyze the performance of NumPy applications using New Relic’s easy to use dashboard
+
+  ## What’s included in this quickstart?
+
+  New Relic's Numpy monitoring quickstart boasts instant full-stack observability out-of-the-box:
+  
+  - Alerts (Adpex score,WSGI Response, transaction error and Requests)
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:28]:
```suggestion
  - Alerts for Apdex score, WSGI response, transaction error, and requests
```

File: quickstarts/python/numpy/config.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  - Easy to use Dashboards (most popular transactions, average transaction duration today compared with that for the previous week, Apdex score comparisons, and more)
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:29]:
```suggestion
  - Easy to use dashboards, including most popular transactions, average transaction duration over time, Apdex score comparisons, and more)
```

File: quickstarts/python/numpy/config.yml [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  - Monitor NumPy Elements and functions
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:30]:
```suggestion
  - Monitor NumPy elements and functions
```

File: quickstarts/python/numpy/config.yml [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+  - Understand infrastructure resource consumption of application performance
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:31]:
```suggestion
  - Understand infrastructure resource consumption
```

File: quickstarts/python/numpy/config.yml [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+  - Alerts that proactively report end users about the quality of their application
+
+summary: |
+  Get a comprehensive understanding of performance with customized dashboards including total requests, total logins, web transactions, and more with New Relic’s NumPy monitoring tool.
```
Comment (akristen) [inline quickstarts/python/numpy/config.yml:35]:
```suggestion
  Get a comprehensive overview of your NumPy app's performance with customized dashboards. Track total requests, total logins, web transactions, and more with New Relic’s NumPy monitoring tool.
```

File: quickstarts/python/numpy/config.yml (hunk 1 above)
Comment (akristen) [inline quickstarts/python/numpy/config.yml:6]:
```suggestion
  ## Quickstart for comprehensive monitoring with NumPy
```

File: dashboards/numpy/numpy.json [hunk 12]
```diff
@@ -0,0 +1,666 @@
+{
```
Comment (aswanson-nr) [inline dashboards/numpy/numpy.json:1]: Let's get the entity GUIDs and explicit times ranges removed from this dashboard!

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Looking like we just need to clean up some of the dashboard NRQL queries

**Comment by pkosanam** [comment]
closing this PR as this quickstart is not present in our current release

**Pull request closed without merging**